python-mini-projects/projects/Create_calculator_app/calculator.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+from tkinter import Tk, END, Entry, N, E, S, W, Button
+from tkinter import font
+from tkinter import Label
+from functools import partial
+
+
+def get_input(entry, argu):
+    entry.insert(END, argu)
+
+
+def backspace(entry):
+    input_len = len(entry.get())
+    entry.delete(input_len - 1)
+
+
+def clear(entry):
+    entry.delete(0, END)
+
+
+def calc(entry):
+    input_info = entry.get()
+    try:
+        output = str(eval(input_info.strip()))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

If you do not have absolute control over the contents of the string passed to the `eval()` function, they might be maliciously used to execute an arbitrary function such as `__import__(&#39;os&#39;).system(&#34;rm -rf $HOME&#34;)`. If the string is expected to only contain literals, consider using the `ast.literal_eval()` method to safely evaluate strings containing Python values from untrusted sources.

[Learn more](https://docs.python.org/3/faq/programming.html#how-do-i-use-strings-to-call-functions-methods)